packages/wdio-sauce-service/src/launcher.js

```diff
@@ -103,6 +103,6 @@ export default class SauceLauncher {
             return
         }
 
-        return new Promise(resolve => this.sauceConnectProcess.close(resolve))
+        return this.sauceConnectProcess.close()
```

Comment:
The resolve was previously passed to the close method, this method no longer expects this value?

Reply:
no, the close method returns a promise so the hook will wait until the tunnel has closed.